src/NuGet.Core/NuGet.Packaging.Core/INuspecCoreReader.cs

```diff
@@ -29,6 +29,11 @@ public interface INuspecCoreReader
         NuGetVersion GetMinClientVersion();
 
         /// <summary>
+        /// Gets zero or more package types from the .nuspec.
+        /// </summary>
+        IEnumerable<PackageType> GetPackageTypes();
```

Comment:
Wouldn't it just be a `ManagedTool, v1.0` at that point? It might be trickier to reason about a collection of `PackageType`s than just look at one.